x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/privileges/40_get_user_privs.yml

```diff
@@ -303,7 +303,7 @@ teardown:
 
   - length: { "applications" : 3 }
   - contains: { "applications" : { "application" : "app-dne", "privileges" : [ "all" ], "resources" : [ "*" ] } }
-  - contains: { "applications" : { "application" : "test-app", "privileges" : [ "user", "dne" ], "resources" : [ "*" ] } }
+  - contains: { "applications" : { "application" : "test-app", "privileges" : [ "dne", "user" ], "resources" : [ "*" ] } }
```

Comment:
If order doesn't matter I've things alphabetically. It destroys the original order in a fairly noticeable way. And it keeps the tests tidy. And it makes it easier for a human to find something in a list. But that is a question for the security folks, I think.

Reply:
I'm actually quite struggling with this test case, because I can't make the test pass even if I change the order of privileges.

Reply:
Not sure why the test always loads `{privileges=[user, dne], application=test-app, resources=[*]}`

Reply:
Alright, it fails on the rest compatibility test. We can mute the test now and then enable it back when the change ported to 7.x